Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Buttons/Krypton.Toolkit.Suite.Extended.Buttons 2019.csproj

```diff
@@ -52,15 +52,14 @@
       <OutputPath>..\..\..\..\Bin\$(configuration)\Main\Krypton.Toolkit.Suite.Extended.Buttons\</OutputPath>
     </PropertyGroup>
     <ItemGroup>
-      <PackageReference Include="System.Management" Version="5.0.0" />
+      <Reference Include="System.Management" />
+      <Reference Include="System.Design" />
```

Comment:
Note: These `System.Design` references will need to be versioned as per the Standard toolkit, in order to allow them to be loaded for .net core projects in the designer

Reply:
@Wagnerp See above

Reply:
@Smurf-IV Do you mean like ```cs <ItemGroup>
        <Reference Include="System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" Condition="$(TargetFramework.StartsWith('net4'))">
            <SpecificVersion>True</SpecificVersion>
            <Version>4.0.0.0</Version>
            <TreatAsUsed>True</TreatAsUsed>
        </Reference>
        <Reference Include="System.Design, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" Condition="$(TargetFramework.StartsWith('net5'))">
            <SpecificVersion>True</SpecificVersion>
            <Version>5.0.0.0</Version>
            <TreatAsUsed>True</TreatAsUsed>
        </Reference>
        <Reference Include="System.Design, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" Condition="$(TargetFramework.StartsWith('net6'))">
            <SpecificVersion>True</SpecificVersion>
            <Version>6.0.0.0</Version>
            <TreatAsUsed>True</TreatAsUsed>
        </Reference>
        <Reference Include="System.Design" Condition="'$(TargetFramework)' == 'net35'" />
    </ItemGroup>```